news/_posts/2020-02-04-the-new-prestashop-test-framework.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: post
+title:  "The (new) PrestaShop Test Framework"
+subtitle: "And how we’re continually improving the quality of the PrestaShop project"
```

Comment:
```suggestion
subtitle: "And how we're continually improving the quality of the PrestaShop project"
```
Regular quote instead of fancy quote

<img src="https://media.tenor.com/images/599f10ed6fbb5792122679ab5470290a/tenor.gif">

Reply:
Looks like the copy paste from Gdocs messed it up